ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/useReportFormValues.tsx

```diff
@@ -138,19 +132,16 @@ const validationSchema = yup.object().shape({
         .array()
         .of(
             yup.object().shape({
-                notifier: yup
-                    .object()
-                    .nullable()
-                    .strict()
-                    .test('is-not-null', 'A notifier is required', (value) => {
```

Comment:
Can you explain the `.nullable()` and `.test()` here? They seem contradictory to me but I'm also not very familiar with yup. I just want to make sure there wasn't a need for this.

Reply:
Good eyes!

For background, compromise neither-v1-nor-v2 `notifier` property:
* if specified: `{ id: string, name: string }`
* if not specified: `null`

The way I interpret original validation:
* `object` and `nullable` tell yup that either an object or `null` are expected values
* `return value !== null && value !== undefined` in `test` function tell yup that a notifier is specified (that is, the property is valid) if not nullish. This is somewhat confusing, because `undefined` means either conversion or form code has messed up.